src/sentry/tasks/relay.py

```diff
@@ -394,3 +397,99 @@ def _schedule_invalidate_project_config(
     projectconfig_debounce_cache.invalidation.debounce(
         organization_id=organization_id, project_id=project_id, public_key=public_key
     )
+
+
+# OpenRouter API endpoint
+OPENROUTER_MODELS_API_URL = "https://openrouter.ai/api/v1/models"
+
+
+@instrumented_task(
+    name="sentry.tasks.relay.fetch_ai_model_costs",
+    queue="relay_config",
```

Comment:
This is not a Relay task, please don't call it Relay and don't put it in the Relay file, it can be its own thing.

This is now a thing that can be used throughout Sentry, it's not Relay specific, Relay just happens to be exposed the data. We're also not going to maintain it in the ingest team.